crypto/src/main/scala/org/alephium/crypto/AES.scala

```diff
@@ -30,7 +30,7 @@ object AES {
   final case class Encrypted(encrypted: ByteString, salt: ByteString, iv: ByteString)
 
   private val saltByteLength       = 64
-  private val ivByteLength         = 64
+  private val ivByteLength         = 12
```

Comment:
does it means current wallet files won't be compatible anymore?

Reply:
I still need to check, not sure so far. It's not so urgent, the risk is low.